charts/open5gs/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v2
-appVersion: "2.4.9"
+appVersion: "2.4.11"
```

Comment:
This is not true if we do not change the image tag in the dependency charts.

Should we add a helper in open5gs charts that creates image tags for dependencies from a global entry in values.yaml?